backstop/tests/carousel.html

```diff
@@ -0,0 +1,355 @@
+              <li>
+                <span class="iui-carousel-navigation-dot iui-second"></span>
+              </li>
+              <li>
+                <span class="iui-carousel-navigation-dot"></span>
+              </li>
+              <li>
+                <span class="iui-carousel-navigation-dot"></span>
+              </li>
+              <li>
+                <span class="iui-carousel-navigation-dot iui-active"></span>
+              </li>
+              <li>
+                <span class="iui-carousel-navigation-dot"></span>
+              </li>
+              <li>
+                <span class="iui-carousel-navigation-dot"></span>
+              </li>
+              <li>
+                <span class="iui-carousel-navigation-dot iui-second"></span>
+              </li>
+              <li>
+                <span class="iui-carousel-navigation-dot iui-first"></span>
+              </li>
+              <li>
+                <span class="iui-carousel-navigation-dot iui-invisible"></span>
+              </li>
+            </ol>
+          </div>
+          <button
+            class="iui-button iui-borderless iui-small iui-carousel-navigation-button"
+            aria-controls="myCarousel-items-3"
+            aria-label="Next slide"
+          >
+            <svg-chevron-right
+              class="iui-button-icon"
+              aria-hidden="true"
+            ></svg-chevron-right>
+          </button>
+        </nav>
+      </div>
+    </section>
+
+    <hr />
+    <h2>Counter</h2>
+    <section id="demo-counter">
+      <div
+        class="iui-carousel"
+        aria-roledescription="carousel"
+        aria-label="Introduction to iTwinUI"
+        aria-live="polite"
+      >
+        <div
+          class="iui-carousel-slider"
+          id="myCarousel-items-2"
+        >
+
+          <div
+            class="iui-carousel-slider-item"
+            style="background-image: url('../assets/image-test.jpg');"
+          >1</div>
+
+          <div
+            class="iui-carousel-slider-item"
+            style="background-color: var(--iui-color-background-4);"
+          >2</div>
+
+          <div
+            class="iui-carousel-slider-item"
+            style="background-color: var(--iui-color-background-5);"
+          >3</div>
+
+        </div>
+
+        <nav class="iui-carousel-navigation">
+          <button
+            class="iui-button iui-borderless iui-small iui-carousel-navigation-button"
+            aria-controls="myCarousel-items-2"
+            aria-label="Previous slide"
+          >
+            <svg-chevron-left
+              class="iui-button-icon"
+              aria-hidden="true"
+            ></svg-chevron-left>
+          </button>
+          <div class="iui-carousel-navigation-container">
+            <div class="iui-carousel-navigation-counter">
+              <span class="iui-carousel-navigation-counter-current">4</span>
+              <span class="iui-carousel-navigation-counter-total">7</span>
```

Comment:
dont have rules for this class.. so maybe not needed? Or we can tackle first-child instead of having current class? Not sure..

Reply:
Removed `.iui-carousel-navigation-counter-total` but left `.iui-carousel-navigation-counter-total` rather than looking for first child incase the user places anything else within the `.iui-carousel-navigation-counter`.